src/webapp/src/App.tsx

```diff
@@ -12,6 +13,7 @@ export const App = () => (
       <Navbar />
       <Switch>
         <Route path='/login' component={AuthenticationOptions} />
+        <Route path='/dashboard' component={Dashboard} />
```

Comment:
I would say this page for me a feed of ads. What you think of renaming this to a little?

Reply:
Perfect, I was missing a name like that. Do you prefer `AdsFeed` or something longer like `FeedOfAds`?

Reply:
perhaps just `feed` might be good

Reply:
> What I refresh the page the login status is being lost. What is your plan in making this persistent?

I implemented something using firebase `auth.onAuthStateChanged`, didn't push it now to keep the commit clean. As soon as we polish what you comment I will show you what I thought of.